### ISSUE TITLE
Support for Installed Applications Flow

### ISSUE THREAD

**Issue by kimar** [jaredhanson/oauth2orize#73]
I didn't find any information about support for installed applications which use redirect_uri=urn:ietf:wg:oauth:2.0:oob.
This currently leads to a redirectal error, which is perfectly correct, but shall lead to a page containing the access_token inside the html body response.

Thanks for your support.

**Comment by kimar** [comment]
What a concidence I'm digging up this old one from its grave exactly two years after you've answered it 😄 

I've implemented the solution and you've described and it works quite well, btw is there any common understanding of how to propagate the `access_token` to the user? I've seen a few ways (e.g. yours using the title, others doing it via JS). I'm currently presenting an "Authenticating…" page to the user and then the app has to decide how to retrieve the token. I've implemented multiple ways.

**Comment by joshperry** [comment]
@kimar This is actually something I yanked from Google, it's not described in the rfc. In fact, it looks like Google is even poo-pooing this method now and recommending it not be used unless necessary (it's hidden beneath an exposer now): https://developers.google.com/identity/protocols/OAuth2InstalledApp#choosingredirecturi

**Comment by kimar** [comment]
I agree, that's definitely what we should be heading for. I've been using this flow for third party apps but I'm about to implement it for our own apps as well and I'm aiming for the custom URI schemes.
